### ISSUE TITLE
Discussion on readings for session 1: Crowdsourcing

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2021-2022#16]
1. Lala Hajibayova & Kiersten F. Latham. 2017. “Exploring Museum Crowdsourcing Projects through Bordieu’s Lens.” _Knowledge Organization_ 44-7, 506-514. Available: https://www.ergon-verlag.de/isko_ko/downloads/ko_44_2017_7_e.pdf
1. Victoria Leonard & Sarah E. Bond. 2019. “Advancing Feminism Online.” _Studies in Late Antiquity_ 3.1, 4–16. Available: https://hcommons.org/deposits/download/hc:23430/CONTENT/victoria-leonard-and-sarah-e.-bond-advancing-feminism-online-sla-2019.pdf/

Please comment on any opinions or responses to these two readings in this ticket, and engage with comments by your colleagues.

**Comment by molmay** [comment]
The Leonard article about crowdsourcing female representation online was by far more accessible than the other article, which I thought was really appropriate for both the description of the projects but also as an attempt to enlist the readership to join the projects. I specifically found the description of crowdsourcing through wikipedia and the WOAH database as being a form of direct action really interesting as I had never thought of editing wikipedia pages (for example) as being quite a radical or progressive thing to do before. 
I did think there could have been some discussion of how digital spaces aren’t always entirely accessible for some people (e.g. people without access to computers or reliable internet connection) because I felt like it was assumed that anybody anywhere could get online and do these things. It would have been interesting, and perhaps in the spirit of accessibility to consider how these resources could be opened up for people that can’t ordinarily get online through the use of communal computers etc in the future.

**Comment by Ghilaevansky** [comment]
> The Leonard article about crowdsourcing female representation online was by far more accessible than the other article, which I thought was really appropriate for both the description of the projects but also as an attempt to enlist the readership to join the projects. I specifically found the description of crowdsourcing through wikipedia and the WOAH database as being a form of direct action really interesting as I had never thought of editing wikipedia pages (for example) as being quite a radical or progressive thing to do before. I did think there could have been some discussion of how digital spaces aren’t always entirely accessible for some people (e.g. people without access to computers or reliable internet connection) because I felt like it was assumed that anybody anywhere could get online and do these things. It would have been interesting, and perhaps in the spirit of accessibility to consider how these resources could be opened up for people that can’t ordinarily get online through the use of communal computers etc in the future.

I agree with you, there should have been some mention of how digital spaces are not always accessible for everyone. Regardless, I liked how the Leonard article described the digital invisibility of women. I would have liked to see some suggestions and specific examples of how the everyday person can combat this inequality, however. The article by Hajibayova was harder to navigate, in my opinion. I didn't feel as though there was a logical flow of information.

**Comment by gabrielbodard** [comment]
>  It would have been interesting, and perhaps in the spirit of accessibility to consider how these resources could be opened up for people that can’t ordinarily get online through the use of communal computers etc in the future.

That's a _really_ good point. I wonder if the WCC have thought about this very much. (I bet they have!) The paper is a couple of years old, which is a lifetime in terms of a fast-moving project like this…

**Comment by Dasha-Barkova** [comment]
“Exploring Museum Crowdsourcing Projects through Bordieu’s Lens” made me think of Hume’s idea of taste. He thought of it as an ability to make moral and aesthetic judgments. In his work, Hume was stressing the fact that taste could only be formed collectively by a group of educated men who agree on what should be called “high” or “low” art. Given that Hume lived in the late 18th century it is obvious that “taste” was created by white, western men which implies classism and colonialism in their perception. Hence why some artifacts and art forms could be considered of a lower status/less important for history (which is again westernized). 

I thought that it is kind of in line with Bourdieu’s links between class and preferences in art, music, film, and theatre. He even summed up his research into a short phrase: “Taste classifies the classifier”.

**Comment by lettychardon** [comment]
“archaeologists, cultural preservationists, curators, and, critically, indigenous people must all interact to influence the selection, acquisition, classification, and presentation of an object.” (Hajibayova & Latham, p.509)

It astonishes me that this is something that still needs to be said. No one can think of everything at once, in this discipline or more generally in life, so different perspectives are essential to be able to take the maximum number of potential issues and points of interest into consideration. Why some people insist on working in isolation on projects with a public aspect is baffling. Non-experts can also have very interesting input, considering that in the case of exhibitions and displays they are often part of the target audience!
